### PR TITLE
Editorial: Clarify the liveness simultaneity requirement

### PR DESCRIPTION
**Pull request by syg**
Taken from Waldemar's suggestion from #185

### CODE REVIEW

File: spec/abstract-jobs.html [hunk 1]
```diff
@@ -222,11 +222,11 @@ <h1>Execution</h1>
           of non-live objects.
         </p>
         <p>
-          Implementations are obligated to empty WeakRefs for all objects in a
-          non-live set simultaneously. In other words, they must not empty a
-          WeakRef pointing to an object _obj_ without emptying out other
-          WeakRefs that, if not emptied, could result in an execution that
-          observes the Object value of _obj_.
+          If an implementation chooses a set _S_ in which to empty WeakRefs, it
```
Comment (bakkot) [inline spec/abstract-jobs.html:225]:
```suggestion
          If an implementation chooses a non-live set _S_ in which to empty WeakRefs, it
```

File: spec/abstract-jobs.html [hunk 2]
```diff
@@ -222,11 +222,11 @@ <h1>Execution</h1>
...
         </p>
         <p>
-          Implementations are obligated to empty WeakRefs for all objects in a
-          non-live set simultaneously. In other words, they must not empty a
-          WeakRef pointing to an object _obj_ without emptying out other
-          WeakRefs that, if not emptied, could result in an execution that
-          observes the Object value of _obj_.
+          If an implementation chooses a non-live set _S_ in which to empty WeakRefs, it
+          must empty WeakRefs for all objects in _S_ simultaneously. In other
+          words, they must not empty a WeakRef pointing to an object _obj_
```
Comment (bakkot) [inline spec/abstract-jobs.html:227]:
```suggestion
          words, an implementation must not empty a WeakRef pointing to an object _obj_
```
(the antecedent is now singular, so "they" is weird)

Comment (ljharb) [reply spec/abstract-jobs.html:227]: singular they is perfectly correct; adding the word “implementation” here triggers the discussion about host vs implementation.

Comment (bakkot) [reply spec/abstract-jobs.html:227]: Implementations are not animate enough to warrant "they". "it", at least.

Comment (ljharb) [reply spec/abstract-jobs.html:227]: Sure, that works fine.

Comment (syg) [reply spec/abstract-jobs.html:227]: I like @bakkot's suggestion. In my preferred taxonomy of host-implementation, implementation is the correct choice. Plus, I already start the paragraph with "If an implementation".

We can debate host-vs-implementation independently. No reason do it in this proposal spec draft.

**Pull request merged**